aten/src/ATen/native/cuda/Blas.cpp

```diff
@@ -887,6 +889,13 @@ _scaled_mm_out_cuda(const Tensor& mat1, const Tensor& mat2,
   TORCH_CHECK(false, "_scaled_mm_out_cuda is not compiled for this platform.");
 #endif
 
+#if defined(USE_ROCM) && ROCM_VERSION >= 60000
+  // rocm's hipblaslt does not yet support amax, so calculate separately
```

Comment:
Are we expecting that hipblaslt will support amax in the near future (although we might use it when the output dtype is bf16/fp16 instead of fp8)?

Reply:
We would need to confirm with PMs, but amax might be planned for ROCm 6.2.  This might not be soon enough for your needs.